### DISCUSSION TITLE
Возможность "раскрашивать" дисциплины

### DISCUSSION THREAD

**Discussion by RomanSoloweow** [0niel/university-app#212] [Идеи]
Опять же по аналогии с "Расписашкой", чтоб можно было помечать лайтовые и не очень пары)

**Comment by 0niel** [comment]
Раскрашивать тип пар, чтобы в календаре кружочки другим цветом отображались?

**Comment by RomanSoloweow** [comment]
Не совсем тип, скорее чтоб можно было свой цвет каждой дисциплине назначить

**Comment by 0niel** [comment]
Не совсем понимаю зачем это нужно? У нас будет задаваться цвет с помощью заметок при их создании, а далее эта заметка будет окрашивать карточку предмета.
![image](https://user-images.githubusercontent.com/51058739/135819819-53ca4b30-b8b4-4c17-af0a-2f33bc295b13.png)

**Comment by RomanSoloweow** [comment]
А эта заметка к конкретной паре или дисциплине в целом)

мб я хочу один раз заметку задать на всю дисциплину типо "ходить всегда" и пометить её красным
